From akpm@linux-foundation.org  Fri Aug  7 16:57:18 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Thu, 06 Aug 2009 16:09:35 -0700
Subject: Staging: udlfb: fix printk format warning
To: greg@kroah.com
Cc: akpm@linux-foundation.org, randy.dunlap@oracle.com
Message-ID: <200908062309.n76N9ZXB011831@imap1.linux-foundation.org>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix printk format warning: use %td for ptrdiff:

drivers/staging/udlfb/udlfb.h:209: warning: format '%d' expects type 'int', but argument 3 has type 'long int'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 drivers/staging/udlfb/udlfb.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/udlfb/udlfb.h
+++ b/drivers/staging/udlfb/udlfb.h
@@ -205,7 +205,7 @@ static int dlfb_set_video_mode(struct dl
 
 			/* send */
 			ret = dlfb_bulk_msg(dev_info, bufptr - dev_info->buf);
-			printk("ret bulk 2: %d %d\n", ret,
+			printk("ret bulk 2: %d %td\n", ret,
 			       bufptr - dev_info->buf);
 
 			/* flush */
